templates/internal/appName/k8s/controller.go.tpl

```diff
@@ -16,10 +16,10 @@ import (
 	"context"
 
 	"github.com/getoutreach/gobox/pkg/log"
-	k8slogging "github.com/getoutreach/services/pkg/k8s/logging"
-	"github.com/getoutreach/services/pkg/k8s/controllers"
-	"github.com/getoutreach/services/pkg/k8s/logging"
-	"github.com/getoutreach/services/pkg/k8s/resources"
+	k8slogging "github.com/getoutreach/k8slib/pkg/k8s/logging"
+	"github.com/getoutreach/k8slib/pkg/k8s/controllers"
+	"github.com/getoutreach/k8slib/pkg/k8s/logging"
```

Comment:
suggestion: line 19 and 21 are pointing to the same thing but with multiple names. That should be consolidated.

Reply:
Looks like `k8slogging` is unused in the template.
```suggestion
	"github.com/getoutreach/k8slib/pkg/k8s/controllers"
	"github.com/getoutreach/k8slib/pkg/k8s/logging"
```